File: Tutorial/CONVERT.cpp
#include <iostream>
#include <string>
using namespace std;
int main()
{
    int n;
    cin >> n;

    for (int i = 1 ; i <= n ; i++)
    {
        double num;
        string kind;
        // cin.ignore(INT_MAX, '\n');
        cin >> num;
        cin.ignore();
        getline(cin, kind,'\n');

        // cout <<"i: " << i << '\n';
        cout << fixed;
        cout.precision(4);

        if (kind == "kg")
            cout << i << ' ' << num * 2.2046 << ' ' << "lb" << '\n';
        else if (kind == "lb")
            cout << i << ' ' << num * 0.4536 << ' ' << "kg" << '\n';
        else if (kind == "l")
            cout << i << ' ' << num * 0.2642 << ' ' << "g" << '\n';
        else if (kind == "g")
            cout << i << ' ' << num * 3.7854 << ' ' << "l" << '\n';
    }
}

File: Tutorial/CONVERT.py
n = int(input())

for i in range(1,n + 1):
    num, kind = map(str, input().split())
    num = float(num)
    if kind == "kg":
        print(i, format(num * 2.2046, ".4f"), "lb")
    elif kind == "lb":
        print(i, format(num * 0.4536, ".4f"), "kg")
    elif kind == "l":
        print(i, format(num * 0.2642, ".4f"), "g")
    elif kind == "g":
        print(i, format(num * 3.7854, ".4f"), "l")